kubejs/server_scripts/base/tags/items/tconstruct/tinkers_anvil.js

```diff
@@ -1,3 +1,3 @@
 onEvent('item.tags', (event) => {
-    event.add('tconstruct:tinkers_anvils', ['tconstruct:scorched_anvil', 'tconstruct:tinkers_anvil']);
```

Comment:
remember to update file placement too please :D

Reply:
Ah, sure thing!
(Thank god everything comes out as I go, because fixing everything again after you tried to fix it is annoying 😆 )